ReferenceTests/src/tests/examples3d.jl

```diff
@@ -527,7 +527,7 @@ end
     # without depth_shift (0f0, red) and one at ∓10eps(1f0) (blue, left/right axis).
     # A negative shift should push the plot in the foreground, positive in the background.
     for (i, _shift) in enumerate((-10eps(1f0), 10eps(1f0)))
-        ax = LScene(fig[1, i], show_axis = false)
```

Comment:
This is a breaking change... With the last release, I intended to make `show_axis` more first class for LScene, now that it's not a property of the scene anymore..Also doesn't really make sense to put this in scenekw anymore, if there are no scene kw for showing an axis.

Reply:
made this an attribute of LScene now